### DISCUSSION TITLE
DUMI_DEV_SERVER_COMPRESS 没有生效

### DISCUSSION THREAD

**Discussion by lch000** [umijs/dumi#2234] [Q&A]
设置了DUMI_DEV_SERVER_COMPRESS 为none，但是在使用中还是发现SSE接口没有分段
![image](https://github.com/user-attachments/assets/4f3c565c-17a4-4e74-a5ca-58be5f4877b3)
<img width="920" alt="image" src="https://github.com/user-attachments/assets/8c0b9b78-e152-451b-84ad-716bf795c228">

**Comment by poozhu** [comment]
这应该是个 bug, 得配置 UMI_DEV_SERVER_COMPRESS 才会生效
